woocommerce/woocommerce-android
# Shipping Labels: add support for shipping items in original packaging

Closes #4003, this adds supporting for moving items to be shipped in their individual original packaging.

<img width="352" alt="Screen Shot 2021-07-08 at 11 25 29" src="https://user-images.githubusercontent.com/1657201/124907306-e3f33500-dfdf-11eb-9e57-d0d6ca693e94.png">   <img width="352" alt="Screen Shot 2021-07-08 at 11 25 13" src="https://user-images.githubusercontent.com/1657201/124907311-e5bcf880-dfdf-11eb-8247-fe289726bac2.png">

#### Testing
1. Make sure dimensions are cleared from [shipping settings](https://d.pr/i/gixQP4) of a  product in your store.
2. Create an order eligible for shipping label creation with this product.
3. Go to order details in the app.
4. Click on Create shipping labels.
5. When you reach the packaging step, click on "Move" button of the item.
6. Select the option "Ship in original packaging"
7. Confirm that there is an error displayed about missing dimensions.
8. Navigate back to the product details, and specify some dimensions in shipping settings.
9. Repeat steps 3 to 6
10. Confirm that the correct dimensions are displayed correctly.
11. Continue the label purchase process.
12. Go to the order details, and check the label details ([the button view details](https://d.pr/i/AmY0CH)), and confirm that it says "individual packaging" 

Update release notes:
